ShoppingCart/ShoppingCart.java

```diff
@@ -0,0 +1,122 @@
+import java.util.*;
```

Comment:
Wild cards shouldn't be used in imports in Java. What issues does it cause can be understood from this article - https://medium.com/@aayushsharda/why-is-it-better-to-avoid-in-import-statements-in-java-ff9ceaca8aeb